src/plugins/fs.c

```diff
@@ -47,6 +47,29 @@
  * A plugin for operations with file systems
  */
 
+const BDFSInfo bd_fs_info[] = {
+    {"xfs", "xfs_db", "xfs_repair", "xfs_growfs", BD_FS_ONLINE_GROW | BD_FS_OFFLINE_GROW},
+    {"ext2", "e2fsck", "e2fsck", "resize2fs", BD_FS_ONLINE_GROW | BD_FS_OFFLINE_GROW | BD_FS_OFFLINE_SHRINK},
+    {"ext3", "e2fsck", "e2fsck", "resize2fs", BD_FS_ONLINE_GROW | BD_FS_OFFLINE_GROW | BD_FS_OFFLINE_SHRINK},
+    {"ext4", "e2fsck", "e2fsck", "resize2fs", BD_FS_ONLINE_GROW | BD_FS_OFFLINE_GROW | BD_FS_OFFLINE_SHRINK},
+    {"vfat", "fsck.vfat", "fsck.vfat", "", BD_FS_OFFLINE_GROW | BD_FS_OFFLINE_SHRINK}
+};
+
+const BDFSInfo *
+bd_fs_get_info (const gchar *type)
```

Comment:
These should be static. And please note that it's not necessary to use the ``bs_fs`` prefix for internal symbols (actually, it's not desirable because using the prefix may lead to a confusion).